types/openseadragon/index.d.ts

```diff
@@ -570,7 +572,10 @@ declare namespace OpenSeadragon {
         );
     }
 
-    class IIIFTileSource extends TileSource {}
+    class IIIFTileSource extends TileSource {
+        constructor(options: TileSourceOptions & {tileFormat?: string});
+        canBeTiled(profile: string[]): boolean;
```

Comment:
I think that one is for tests only and only if used via special property:
https://github.com/openseadragon/openseadragon/blob/898b8456d1c938f38e86fe40efe5effee3e40b7c/test/modules/iiif.js#L175

Reply:
You're right, thanks!